src/registry_schemas/schemas/agm_location_change.json

```diff
@@ -12,15 +12,21 @@
             "type": "object",
             "required": [
                 "year",
-                "newAgmLocation"
+                "reason",
+                "agmLocation"
             ],
             "properties": {
                 "year": {
                     "type": "string",
                     "description": "Year of the AGM, Must be on or after incorporation year and cannot be future year."
                 },
-                "newAgmLocation": {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                "reason":{
+                    "type": "string",
+                    "maxLength": 2000
+                },
+                "agmLocation": {
```

Comment:
this doesn't make it easy to support the business rules around location and approvals.
how does this support the requirements?

Reply:
Thor - as per our conversation, yes, this would make it hard to determine out of country and maybe it makes sense to go back to using the address component.  

Will need to get a decision on this from the business.  @chenhongjing can you follow up with Mihai on this?